example/mutatingwebhook.go

```diff
@@ -57,15 +54,6 @@ func (a *podAnnotator) Handle(ctx context.Context, req admission.Request) admiss
 	return admission.PatchResponseFromRaw(req.AdmissionRequest.Object.Raw, marshaledPod)
 }
 
-// mutatePodsFn add an annotation to the given pod
-func (a *podAnnotator) mutatePodsFn(ctx context.Context, pod *corev1.Pod) error {
```

Comment:
The intend of having this method is that in most cases the user's business logic should be contained in this method. Anything else in `Handle()` method should be unchanged.
So IMO it's better to keep it.

Reply:
IMO, it makes the example harder to follow, which is why I removed it.